blog-posts/middleware-on-redux/middleware-on-redux.md

```diff
@@ -0,0 +1,81 @@
+---
+published: false
+title: 'Middleware on redux'
+cover_image: 'https://raw.githubusercontent.com/ramclen/documenting-dev/master/blog-posts/middleware-on-redux/assets/redux.png'
+description: 'Answering questions about middleware on redux'
+tags: redux, middleware, typescript, javascript
+series:
+canonical_url:
+---
+
+If you are start learning Redux, middlewares could be a pain know what it is and why we need them... So I am going to try to give you few keys (and links) to help you in your learning process.
```

Comment:
I would use "If you are starting learning" or "starting to learn". The second one sounds better.